README.md

```diff
@@ -1,24 +1,5 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+<img src="https://travis-ci.org/Lockeriano/CookBook.svg?branch=master" />
+<a href="https://codeclimate.com/github/Lockeriano/CookBook/maintainability"><img src="https://api.codeclimate.com/v1/badges/da3fedd9b0ee649899d9/maintainability" /></a>
+<a href="https://codeclimate.com/github/Lockeriano/CookBook/test_coverage"><img src="https://api.codeclimate.com/v1/badges/da3fedd9b0ee649899d9/test_coverage" /></a>
```

Comment:
We can use Markdown features here and the `Issues Count` badge is missing. Below is example:

```
[![Build Status](https://travis-ci.org/distribusion/supa.svg?branch=master)](https://travis-ci.org/distribusion/supa)
 [![Code Climate](https://codeclimate.com/repos/587387071c36ea7203000e0d/badges/19b714c64bf6f028a58c/gpa.svg)](https://codeclimate.com/repos/587387071c36ea7203000e0d/feed)
 [![Test Coverage](https://codeclimate.com/repos/587387071c36ea7203000e0d/badges/19b714c64bf6f028a58c/coverage.svg)](https://codeclimate.com/repos/587387071c36ea7203000e0d/coverage)
 [![Issue Count](https://codeclimate.com/repos/587387071c36ea7203000e0d/badges/19b714c64bf6f028a58c/issue_count.svg)](https://codeclimate.com/repos/587387071c36ea7203000e0d/feed)
```